samples/apps/passport/next-connect-kit/README.md

```diff
@@ -0,0 +1,22 @@
+# POC
+
+**Step 1**
+
+Create a Passport application on Immutable Hub by following [these simple steps](https://docs.immutable.com/docs/zkEVM/products/passport/setup)
+
+**Step 2**
+
+Update `PUBLISHABLE_KEY` and `CLIENT_ID` in `src/app/passport.ts`
+
+```ts
+// Immutable Publishable Key (you will find it in the Immutable Hub in the `API Keys` section)
+const PUBLISHABLE_KEY = 'PUBLISHABLE_KEY';
+// Hub Publishable Key (you will find it in the `Passport` section)
+const CLIENT_ID = 'CLIENT_ID'; 
+```
+
+**Step 3**
+
+```sh
+yarn && yarn dev
+```
```

Comment:
Was it possible to update the READMEs to the new formatting that the repo will follow now? Currently the monorepo README and the packages READMEs are all being updated to a similar, consistent format for easy to consume information and some more clarity. You can take a look at the README in the blockchain-data package for eg [here](https://github.com/immutable/ts-immutable-sdk/blob/main/packages/blockchain-data/sdk/README.md).

Just as an example for this README, maybe something like the below, that can then be applied to the other READMEs?

-----

# Next Connect Kit

### Pre-requisites

Install dependencies for the sample app

```bash
yarn install
```

### Running the app

**Step 1**

Create a Passport application on Immutable Hub by following [these simple steps](https://docs.immutable.com/docs/zkEVM/products/passport/setup)

**Step 2**

Update `PUBLISHABLE_KEY` and `CLIENT_ID` in `src/app/passport.ts`

```typescript
// Immutable Publishable Key (you will find it in the Immutable Hub in the `API Keys` section)
const PUBLISHABLE_KEY = 'PUBLISHABLE_KEY';
// Hub Publishable Key (you will find it in the `Passport` section)
const CLIENT_ID = 'CLIENT_ID'; 
```

**Step 3**

Run the Connect Kit sample app in dev mode

```bash
yarn dev
```